dss-api.yml

```diff
@@ -1385,7 +1385,7 @@ paths:
                     name:
                       description: Name of the file.
```

Comment:
Consider briefly summarizing usage of slashes here.

Reply:
Good call.  Adding it in.